### ISSUE TITLE
Check if we're GDPR compliant

### ISSUE THREAD

**Issue by razzeee** [flathub/frontend#90]

**Comment by nedrichards** [comment]
How would we not be? Who is responsible for doing this? (it's going to be some subset of me, @barthalion @ramcq etc.) I assume that we consider this to be a release blocker for the new frontend?

**Comment by razzeee** [comment]
I'm just not very familiar with the topic, but if the old page was compliant, the new one is probably too. I just figured it might be good to check again.

Matomo also honors the do net track flag, which is probably all we need to do.

**Comment by nedrichards** [comment]
Right. I will ensure that this gets looked at to make sure we're still happy.

**Comment by razzeee** [comment]
Loading fonts via google might be something we want to change for e.g. not sure if that's strictly GDPR compliance.

**Comment by razzeee** [comment]
Font issue has been addressed now
